HTML Folder/AboutMe.html

```diff
@@ -0,0 +1,70 @@
+                            <a href="../Index.html">Home 🡓</a>
+                            <ul>
+                                <li><a href="../Index.html#welcometoflicksight">Welcome to FlickSight</a></li>
+                                <li><a href="../Index.html#visionandvalues">Vision and Values</a></li>
+                                <li><a href="../Index.html#features">Features</a></li>
+                                <li><a href="../Index.html#oursolution">Our Solution</a></li>
+                                <li><a href="../Index.html#contact">Contact</a></li>
+                            </ul>
+                        </li>
+                        <li>
+                            <a href="../HTML Folder\ContactUs.html">Contact Us</a>
+                        </li>
+                        <li>
+                            <a href="../HTML Folder\AboutMe.html">About Me</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+
+            <main>
+                <div class="aboutmecontainer">
+                    <img src="../Media\Photos\Profilephoto_AnaCarmona.JPG" alt="Ana'sProfilePhoto">
+                        <p>
+                            <strong>About Me:</strong>
+                            Hey there! My name is Ana, I'm a Customer Success Specialist, aspiring <em>Developer</em> and Project Manager. 
+                            <br></br>
+                            I'm a driven, proactive, and organized individual with a strong background in administrative tasks, project coordination, sales,
+                            and everything that comprises customer success strategies and user experience. As a result, I have developed excellent analytical,
+                            communication, and problem-solving skills. 
+                            I'm also passionate about optimizing and improving business processes and customer satisfaction, using research and critical thinking 
+                            to find ideal solutions.  
+                            <br></br>
+                            In my free time, I enjoy reading, learning, creating, playing guitar and watching tv shows and movies.
+                        </p>
+                    </div>
+
+                <div id="contactinfo">
+                    <h2>Let's connect!</h2>
+                    <div class="contactlinks">
+                        <a href="https://www.linkedin.com/in/anacarmona98/" target="_blank"><img src="../Media\Photos\linkedin_black_icon_73508.png" width="65" height="65"></img>
+                        <a href="mailto:amcr981@gmail.com" target="_blank"><img src="../Media\Photos\email_black_icon.png" width="75" height="65"></img></a>
+                        <a href="https://github.com/anacarmona14" target="_blank"><img src="../Media\Photos\github_black_icon.png"  width="70" height="75"></img></a>
+                </div>
+            </main>
+
+            <br></br>
```

Comment:
avoid using <br> tag for spacing, you can achieve the same thing with css margin/padding and your html code will look cleaner